third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -2240,6 +2297,76 @@ def join(self, other, *, on: Optional[str] = None, how: str) -> DataFrame:
 
         Join columns with `other` DataFrame on index
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+        Join two DataFrames by specifying how to handle the operation:
+
+            >>> df1 = bpd.DataFrame({'col1': ['foo', 'bar'], 'col2': [1, 2]}, index=[10, 11])
+            >>> df1
+               col1  col2
+            10  foo     1
+            11  bar     2
+            <BLANKLINE>
+            [2 rows x 2 columns]
+
+            >>> df2 = bpd.DataFrame({'col3': ['foo', 'baz'], 'col4': [3, 4]}, index=[21, 22])
```

Comment:
Could we please have at least one common key, e.g. have `index=[11, 22]` here? `11` would showcase matching rows nicely in the left, right and inner joins

Reply:
Good suggestion. Done.